proto/scheduler.proto

```diff
@@ -24,13 +24,21 @@ message LeaseTaskRequest {
   // Mutually exclusive with `release`.
   bool finalize = 2;
 
+  // DEPRECATED
```

Comment:
In a followup can we remove and mark this field as reserved?

Reply:
We can't do this just yet because of user owned runners. I will go check if people are upgrading their executors.